test/SlowTests/Issues/RavenDB_13178.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Server.Config;
+using Xunit;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_13178 : RavenTestBase
+    {
+        [Fact]
+        public void CompilationOfJavaScriptIndexShouldNotTakeIntoAccountTheMaxStepsForScript()
+        {
+            using (var store = GetDocumentStore(new Options
+            {
+                ModifyDatabaseRecord = record => record.Settings[RavenConfiguration.GetKey(x => x.Indexing.MaxStepsForScript)] = "1"
+            }))
+            {
+                new UsersByPhones().Execute(store);
```

Comment:
Does it fail without the change?
I don't see any documents here.

Reply:
Yes, it fails during JS compilation in JavaScriptIndex ctor